### PR TITLE
csvtranspose

### PR DESCRIPTION
**Pull request by Fil**
csvtranspose with python (see also #221)

not battle-tested with enormous files, but it works on a 1.5MB file I had
Also I've checked that `csvtranspose|csvtranspose` is identity

### CODE REVIEW

**Comment by Fil** [comment]
travis-CI wants this to be ported to python3 (help wanted)

File: csvkit/utilities/csvtranspose.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        pass
+
+    def main(self):
+        self.args.columns = None
+        self.args.no_header_row = True
+
+        rows, column_names, column_ids = self.get_rows_and_column_names_and_column_ids(**self.reader_kwargs)
+        rows = map(list, zip(*rows))
+
+        column_names = list(rows[0])
```
Comment (jpmckinney) [inline csvkit/utilities/csvtranspose.py]: I think you mean `list(rows)[0]` ?

Comment (Fil) [inline csvkit/utilities/csvtranspose.py]: no: the contents of the first column of the original csv become the names of the columns in the transposed csv

**Comment by jpmckinney** [comment]
Thank you for preparing this new CSV tool. However, the maintainers have decided to not merge and maintain new tools; there is simply not enough time to maintain new tools. Our focus is instead on making the existing tools as good as possible.

We encourage you to release and maintain your tool as a separate Python package. You may want to use the agate library, which csvkit uses for most of its CSV reading and writing. Doing so will make it easier to maintain common behavior with csvkit’s tools.

**Pull request closed without merging**